legacy/ui/legacy/src/main/java/com/fsck/k9/ui/settings/SettingsListFragment.kt

```diff
@@ -131,6 +136,20 @@ class SettingsListFragment : Fragment(), ItemTouchCallback {
                     url = getString(R.string.user_forum_url),
                     icon = Icons.Outlined.Help,
                 )
+
+                if (fundingManager.isFundingFeatureIncluded()) {
+                    addIntent(
+                        text = getString(R.string.settings_list_action_support, appNameProvider.appName),
+                        icon = Icons.Outlined.Favorite,
+                        intent = FeatureLauncherActivity.getIntent(requireActivity(), FeatureLauncherTarget.Funding),
+                    )
+                } else {
+                    addUrlAction(
+                        text = getString(R.string.settings_list_action_support, appNameProvider.appName),
+                        url = getString(R.string.donate_url),
+                        icon = Icons.Outlined.Favorite,
+                    )
```

Comment:
This would include a donation link in K-9 Mail. Is that the intended behavior?

Reply:
I think we need a third option to place the donation link only for TB foss and K-9 should use the no-op version. K-9 for F-Droid is still in discussion.

Reply:
Added a new donationlink variant to distinguish between no-op and url.